cloudservice/datadog/datadog_test.go

```diff
@@ -0,0 +1,123 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package datadog
```

Comment:
Is it normal to don't add the **_test** suffix?

Reply:
huh, I see it's totally inconsistent across our codebase. Let's fix that in a separate PR?